libs/common/src/sql-db-entities/facility-utilisation-data/facility-utilisation-data.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { Entity, OneToMany, PrimaryColumn } from 'typeorm';
+import { AuditableBaseEntity } from '../base-entities';
+import { FeeRecordEntity } from '../fee-record';
+import { MonetaryColumn } from '../custom-columns';
+
+@Entity('FacilityUtilisationData')
+export class FacilityUtilisationDataEntity extends AuditableBaseEntity {
```

Comment:
After thinking a bit about this, I think it might be useful for us to store report period on this as well so that we know what month/time this data is for. We could assume it's always for the most recent report period behind the one we're doing but it would also make our data in the data lake etc. make more sense downstream.
What do you think?